src/index.js

```diff
@@ -47,14 +47,17 @@ class GifPlayerContainer extends React.Component {
     const newGif = newProps.gif;
     const oldStill = oldProps.still;
     const newStill = newProps.still;
-    if (oldGif === newGif && oldStill === newStill) {
+    const oldAutoPlay = oldProps.autoPlay;
+    const newAutoPlay = newProps.autoPlay;
+    if (oldGif === newGif && oldStill === newStill && oldAutoPlay === newAutoPlay) {
```

Comment:
I think we can revert the new code here - we don't want to change the playing state unless `gif` is different.